project/pimcore.py

```diff
@@ -0,0 +1,14 @@
+from .remote import RemoteProject
+
+
+class Pimcore5(RemoteProject):
+    major_version = 'v5.8.2'
+    remote = 'https://github.com/pimcore/pimcore.git'
+
+    @property
+    def platformify(self):
+        return ['cd {0} && rm -rf .ci .travis .gitlab-ci.yml .travis.yml .github CONTRIBUTING.md'.format(self.builddir),
+                'cd {0} && COMPOSER_MEMORY_LIMIT=-1 composer create-project pimcore/skeleton tmp_build'.format(self.builddir),
+                'cp -r {0}tmp_build/* {0} && rm -r {0}tmp_build'.format(self.builddir) 
+                ] + super(Pimcore5, self).platformify + [
+                'cd {0} && composer update --no-scripts && composer require platformsh/config-reader doctrine/orm --no-scripts'.format(self.builddir),]
```

Comment:
Interesting.  This would be the only time we put the parent commands in the middle of the series.  Why?

Reply:
Is doctrine/orm not included in the skeleton out of the box?  Why is that a separate require?

Reply:
Doctrine is not included and it's needed for the deploy hook when updating the database encoding to the Picmore supported format.

Reply:
So the Pimcore skeleton doesn't include Doctrine, but it requires doctrine?  I don't get it.

Reply:
So to clarify, the application by itself doesn't require the doctrine/orm but requires utf8mb4 database charactet_set , so I added the doctrine/orm so I can modify the database character_set in the deploy hook, I could install the CLI and do it through the CLI too. Is there a way to do this through the service.yaml file, if not this could be handy feature to add.

Reply:
If it's not using Doctrine post-install, using Doctrine just to issue a call against the database is way overkill.  Especially ORM, which is the full object relational mapper.  If this is just for your own script then just use PDO directly, no dependency at all.

Although it's still an upstream bug if this is required at all and Pimcore needs to fix that.